twins/api/http/endpoint_twins_test.go

```diff
@@ -184,11 +191,14 @@ func TestAddTwin(t *testing.T) {
 			auth:        token,
 			status:      http.StatusBadRequest,
 			location:    "",
+			err:         svcerr.ErrMalformedEntity,
 		},
 	}
 
 	for _, tc := range cases {
 		repoCall := auth.On("Identify", mock.Anything, &magistrala.IdentityReq{Token: tc.auth}).Return(&magistrala.IdentityRes{Id: testsutil.GenerateUUID(t)}, nil)
+		repoCall1 := twinRepo.On("Save", mock.Anything, mock.Anything).Return(retained, tc.err)
```

Comment:
Try and reduce the usage of `mock.Anything`. This applies to other PRs and also other cases in this PR